content/posts/irods-development-update-september-2023.md

```diff
@@ -0,0 +1,258 @@
+Title: iRODS Development Update: September 2023
+Date: 2023-09-26 17:00
+Author: Kory Draughn
+Slug: irods-development-update-september-2023
+Status: published
+
+
+Hello Everyone,
+
+To start this month's update, let's talk about two new API endpoints that will be provided by iRODS 4.3.1. The first API is called `rc_get_resource_info_for_operation`. It is designed as a more flexible alternative to `rcGetHostForPut` and `rcGetHostForGet`. The problem with those APIs is that they only return the hostname of the server holding the storage. Any hierarchy information obtained on the server-side is thrown away. The new API improves upon the old APIs by returning the hostname and the resource hiearchy. The second API is `rc_get_library_features`. This API allows non-C/C++ clients to query the connected server for supported features. The goal of this is to give clients a proper way to detect features without needing to check version numbers. The C++ connection pool library has grown new connection refresh options. These options are important for C++ clients which run as servers (e.g. the HTTP API). We've improved the delay server migration logic by making it more difficult for there to be two or more delay servers running within the same zone. Here's a big one, we're planning to make the iRODS 4.4 compile against libstdc++ by default. libstdc++ is used by the majority of Unix-like OS. This change will help with making iRODS more approachable for people who have to compile from source, among other things.
```

Comment:
hiearchy -> heirarchy

Reply:
i think you could break this paragraph up a bit, but not required.

Reply:
"by making it more difficult" -> "impossible" ?    a bolder claim, but... it really would be quite hard to have two at this point...

Reply:
"the iRODS 4.4" -> "iRODS 4.4". ...   and this should ... definitely be its own paragraph/idea.

Reply:
Done.